src/components/LaunchFilter/LaunchFilter.scss

```diff
@@ -0,0 +1,7 @@
+.launch-filter-list {
+  position: fixed;
+  z-index: 10;
```

Comment:
Why 10?

Reply:
It's more than 1 (z-index of Slider container)

Reply:
change to the nearest one to overlap

Reply:
change to 2